internal/etcd/etcd.go

```diff
@@ -149,6 +151,74 @@ func (c *ClientEtcdAPI) Close() error {
 	return nil
 }
 
+// UsedSpacePercentage returns the used space as a percentage of the storage quota
+func (c *ClientEtcdAPI) UsedSpacePercentage(ctx context.Context, member Member) (int, error) {
+	const (
```

Comment:
Do we need these at this point of code?

Reply:
I've moved this to be closer to where they've been used